ddmrp_cron_actions_as_job/models/stock_buffer.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Camptocamp (https://www.camptocamp.com)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import models
+
+from odoo.addons.queue_job.job import identity_exact, job_auto_delay
```

Comment:
No need to protect this import?

Reply:
I thought it was no longer required in 13.0? I'll check

Reply:
After some tests, I confirm it's no longer needed

Reply:
Didn't know, thanks!